patches/numpy-1.26.4.patch

```diff
@@ -0,0 +1,12 @@
+*** numpy-1.26.4.old/pyproject.toml	2024-03-28 10:10:32.690436902 -0400
```

Comment:
Good practice for these sort of patches would be to link to the upstream PR where you've sent the patch upstream - the expectation would be that the patch would be a fix that would go away when it's fixed upstream